scripts/test-sim-benchmark.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -ueo pipefail
+
+# Purpose: This script executes uplink upload and download benchmark tests against storj-sim.
+# Setup: Setup and run storj-sim. Remove existing uplink configs.
+# Usage: from the root of storj dir, run ./scripts/test-sim-benchmark.sh
```

Comment:
hmm good question.  with storj-sim, the satellite public grpc addr **is** available as an env var, but the console addr is not. So that means in the current state, the regEndpoint would still need to be hardcoded. 
However, I agree with you and that is less than ideal. Let me think about other possible solutions so that we can use the `storj-sim network test `, that feels like the right thing here.

Reply:
It ought to be simple to add the console address to the things which are exported to the environment. You can see how it might be done at https://github.com/storj/storj/blob/b0ca12e31eef8922a68b79ff1ffe46a5d637255e/cmd/storj-sim/network.go#L384 .

Reply:
using `storj-sim network test` turned out to make this much easier since I could follow the common pattern of using the `GATEWAY_0_API_KEY`.  Using that existing api key, made it so I didn't have to generate a new api key and made this script much simpler.